scripts/20-kansas.sh

```diff
@@ -32,6 +32,13 @@ ogr2ogr -sql "SELECT '2016' AS year, 'Kansas' AS state, 'Sedgwick' as county, PR
         -t_srs EPSG:4326 -f GPKG -nln state -append \
         $TEMPDIR/state.gpkg data/20-kansas/2016/20173-sedgwick/precincts.geojson
 
+#
+# Add Shawnee County (FIPS 227) to the statewide Geopackage file.
+#
+ogr2ogr -sql "SELECT '2016' AS year, 'Kansas' AS state, 'Shawnee' as county, PRECINCT AS precinct, 'polygon' AS accuracy FROM OGRGeoJSON" \
```

Comment:
I think `'Shawnee'` here should be `227` instead.

Reply:
Is that true just for that particular line? All the other county-specific commands above it use the county name, not the FIPS code, for the sql statement.